opencti-platform/opencti-graphql/src/domain/backgroundTask.js

```diff
@@ -44,20 +45,22 @@ export const findAll = (context, user, args) => {
   return listEntities(context, user, [ENTITY_TYPE_BACKGROUND_TASK], args);
 };
 
-const buildQueryFilters = async (rawFilters, search, taskPosition) => {
+const buildQueryFilters = async (user, filters, search, taskPosition) => {
+  const inputFilters = filters ? JSON.parse(filters) : undefined;
+  const finalFilters = adaptFiltersWithUserConfidence(user, inputFilters);
```

Comment:
this produces a filter on user's confidence AND the initial task filters.
No error would be produced in that case...  the task would complete with success (unless we have other way to report it ?)